posthog/internal_metrics/team.py

```diff
@@ -0,0 +1,301 @@
+import json
+import secrets
+from functools import lru_cache
+from typing import Dict, Optional
+
+from django.conf import settings
+from sentry_sdk.api import capture_exception
+
+from posthog.models.dashboard import Dashboard
+from posthog.models.dashboard_item import DashboardItem
+
+NAME = "Posthog Internal Metrics"
```

Comment:
Since this is a "fake" org iiuc, would it make sense to prefix with "__" or something similar, if we in the future create more "fake" orgs for whatever reason, then it would be easy to distinguish them from real usage.

Reply:
IMO the way to consistently distinguish them is via the flag on the model (for_internal_metrics) - the name is kind of secondary to that.